Gemfile

```diff
@@ -25,13 +25,9 @@ group :development, :test do
   gem 'serverspec',              :require => false
   gem 'puppet-lint',             :require => false
   gem 'pry',                     :require => false
+  gem 'pry-byebug',              :require => false
```

Comment:
Typically used for ruby 2 only. Most folks create a Gemfile.local with this stuff in it. I have a selection based on which version of ruby I'm using but I guess it doesn't make a difference in about a month, eh?